x/leverage/simulation/operations.go

```diff
@@ -211,13 +215,14 @@ func SimulateMsgSetCollateralSetting(
 		r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context,
 		accs []simtypes.Account, chainID string,
 	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
-		from, coin, skip := randomCollateralFields(r, ctx, accs, lk)
+		from, token, skip := randomSpendableFields(r, ctx, accs, bk)
```

Comment:
We can generate the denom here using `lk.FromTokenToUTokenDenom` on a random `Token.BaseDenom` from `lk.GetAllTegisteredTokens`.

In that way, we are selecting from all possible uToken denominations.